application/service/person.go

```diff
@@ -33,8 +34,26 @@ func (service PersonService) getCacheKey(cacheKeyType string, value string) stri
 }
 
 func (service PersonService) Create(contextControl domain.ContextControl, person domain.PersonDomain) (domain.PersonDomain, error) {
+	personCopy := person
+	typePerson := personCopy.Tipo_pessoa
```

Comment:
Qual o motivo deste ponto? Qual o motivo de usar o personCopy ao invés de person diretamente?

Reply:
Ajustado.

Reply:
Entendi. Mas aí que está uma das questões de se trabalhar numa arquitetura hexagonal ou clean code, quem for usar ou for usado pelo "service" é quem tem que se preocupar ao manipular o objeto de domain que o "service" consume e produz, ou seja, quem usa ou é usado pelo "service" que se preocupa com o domain. Capitou?

Reply:
Sim, entendi. Tinha feito tudo isso anteriormente exatamente pra não entrar com formatação no DB. Já tá ajustada essa parte nesse ultimo commit que fiz

Reply:
Essa remoção dos caracteres pode ficar dentro do output de DB, pois é uma característica do DB salvar sem caracteres non numericos